pages/installing/gettingStarted-Humming.md

```diff
@@ -0,0 +1,217 @@
+
+Locate the .img file in your Downloads folder. It should be named something like `resin-myApp-0.1.0-0.0.14.img`, now click "choose".
+
+You can now insert your microSD card into your host machine and click continue. PiFiller will look for your SD card and tell you when it finds it.
+
+__NOTE:__ make 100% sure that the SD card it finds is in fact the correct card.
+
+Click continue and piFiller will write the SD card. This can 5-25 minutes depending on your machine. Once this is done, skip down to [setting up your device](/#/pages/installing/gettingStarted-Humming.md#setting-up-your-device)
+
+### Windows
+
+To burn OS images to SD cards on windows, you will need to install [Win32 disk imager][win32-disk-imager]. Once you download it, you can launch win32 disk imager by clicking on the "Win32DiskImager" file in the folder that you extracted it to.
+
+Now in Win32DiskImager, click on the folder icon to select which `.img` file you wish to burn. A file browser window will open and you will need to select your hummingBoard image from the Downloads folder. It should be the extracted version and named something like this `resin-myApp-0.1.0-0.0.14.img`.
+
+Next insert your SD card into your host computer and in the Win32DiskImager GUI, select your SD card when it appears.
+
+__NOTE:__ Be very careful to make sure that you have selected the right SD card. Double check this!! Otherwise you could end up writing over your host machines harddisk.
+
+Once you have made your selections and are 100% sure you are writing to your SD card and nothing else, you can click write and wait for the SD card to be burnt.
+
+Once it is completed, you can carry on setting up your device as shown below.
+
+## Setting Up Your Device
+
+Put the SD card into your device, and connect either the ethernet cable or WiFi adapter. Now power up the device by inserting the micro usb cable.
+
+It can take a few minutes for the device to boot up and appear on the dashboard, so grab some tea while you wait.
+
+<!-- TODO: get insert gif for hummingboard -->
+<!-- ![insert SD](/img/BBB/sd_card_BBB.jpg) -->
+
+While you wait resin.io is partitioning your SD card, installing a custom linux environment and establishing a secure connection with our servers.
+
+If you have a class 10 SD card and a fast internet connection your device should appear on the dashboard in around 7 minutes. Note that Class 4 SD cards can take up to 3 times longer so it's well worth investing in the fastest card you
+can find.
+
+## Running Code On Your Device
+
+![git pushing](/img/screenshots/git_pushing.png)
+
+<!-- TODO: switch example_app link to point to right app. -->
+A good little project to get you started is this [simple webserver][example_app_node] written in node.js. It will respond with `Hello, World!` on port `:80`.
+
+To clone it, run the following in a terminal:-
+```
+git clone git@github.com:resin-projects/simple-server-node.git
```

Comment:
I think it's better to use the https method if you're an anonymous user/do not have a public key setup?  At least it's the url github gives me when visiting a repository I don't have write access to